index.js

```diff
@@ -422,3 +424,15 @@ class Event {
     }
   }
 }
+
+class Session {
+  constructor (listeners) {
+    this.listeners = listeners;
+  }
+
+  stop() {
```

Comment:
```suggestion
  stop () {
```

(Seeing as we're following semistandard in this project.)